taxonomy/models.py

```diff
@@ -0,0 +1,31 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+
+class Vocabulary(models.Model):
+    repository = models.ForeignKey("Repository")
+    name = models.TextField()
```

Comment:
From the [TextField](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.TextField) docs, it looks like max_length is enforced only in the UI for TextField (since Django 1.7). It is required for CharField though.

In MySQL there's the VARCHAR type which stores text inline and is indexable and the TEXT type which is somewhat out of band and is not indexable, but is able to store larger amounts of text (> 64k). Does this translate to postgres? Any insight on whether I should use CharField or TextField in general? I'm guessing CharField should be used here if postgres has similar ideas about storing text